src/RA_AchievementOverlay.cpp

```diff
@@ -636,41 +636,39 @@ void AchievementOverlay::DrawFriendsPage(HDC hDC, int nDX, _UNUSED int, const RE
 
 void AchievementOverlay::DrawAchievementExaminePage(HDC hDC, int nDX, _UNUSED int, _UNUSED const RECT&) const
 {
-    char buffer[256];
+    const auto nNumAchievements = g_pActiveAchievements->NumAchievements();
 
-    const unsigned int nNumAchievements = g_pActiveAchievements->NumAchievements();
+    constexpr auto nAchievementStartX = 0;
+    constexpr auto nAchievementStartY = 80;
 
-    const int nAchievementStartX = 0;
-    const int nAchievementStartY = 80;
+    constexpr auto nLoadingMessageX = 120;
+    constexpr auto nLoadingMessageY = 300;
 
-    const int nLoadingMessageX = 120;
-    const int nLoadingMessageY = 300;
-
-    const int nCoreDetailsY = 150;
-    const int nCoreDetailsSpacing = 32;
+    constexpr auto nCoreDetailsY = 150;
+    constexpr auto nCoreDetailsSpacing = 32;
 
-    const int nWonByPlayerYOffs = 300;
-    const int nWonByPlayerYSpacing = 28;
-
-    const int nRecentWinnersSubtitleX = 20;
-    const int nRecentWinnersSubtitleY = 250;
+    constexpr auto nWonByPlayerYOffs = 300;
+    constexpr auto nWonByPlayerYSpacing = 28;
 
-    const int nWonByPlayerNameX = 20;
-    const int nWonByPlayerDateX = 220;
+    constexpr auto nRecentWinnersSubtitleX = 20;
+    constexpr auto nRecentWinnersSubtitleY = 250;
 
-    char bufTime[256];
+    constexpr auto nWonByPlayerNameX = 20;
+    constexpr auto nWonByPlayerDateX = 220;
 
     const auto pAch =
         gsl::make_not_null<const Achievement*>(&g_pActiveAchievements->GetAchievement(m_nAchievementsSelectedItem));
 
-    const time_t tCreated = pAch->CreatedDate();
-    const time_t tModified = pAch->ModifiedDate();
+    const auto tCreated = pAch->CreatedDate();
+    const auto tModified = pAch->ModifiedDate();
 
     DrawAchievement(hDC, pAch, nDX + nAchievementStartX, nAchievementStartY, TRUE, FALSE);
 
-    if (m_nAchievementsSelectedItem >= (int)nNumAchievements)
+    if (m_nAchievementsSelectedItem >= ra::to_signed(nNumAchievements))
         return;
 
+    char bufTime[256]{};
+    char buffer[256]{};
```

Comment:
#221 Has stuff I was trying to do here so instead of doing that, I just initialized the array (ensures null-termination)